web-api/workflow-terraform/migration/main/lambdas/migration-segments.js

```diff
@@ -42,8 +42,6 @@ const scanTableSegment = async (
     await dynamoDbDocumentClient
       .scan({
         ExclusiveStartKey: lastKey,
-        ExpressionAttributeValues: { ':prefix': 'stream-event-id' },
-        FilterExpression: 'NOT begins_with(pk, :prefix)',
```

Comment:
This code ensures that existing `stream-event-id` records are not migrated when we perform a migration. We don't want or need these records anymore. Should we wait to remove this code until we've done a migration in every environment?

Reply:
dum question: what's the difference between removing this code before we run the migrations vs. removing this code after?

Reply:
The purpose of this code is to _not migrate_ `stream-event-id` records.

Helpful context: for the first several months that the `stream-event-id` records were being generated, they were generated with TTL values in milliseconds (even though they're supposed to be seconds). We haven't run a migration in prod since then, so there are millions of `stream-event-id` records with TTLs that won't occur for 50,000 years.

So if we take this code out now, the `stream-event-id` records will continue to exist because without those lines, they'll get migrated.

Or we can wait to take this code out until we've run a migration in every environment.

Reply:
I think it's safe, and important, to keep until we've run a migration.